src/ontology/mondo-edit.obo

```diff
@@ -420458,7 +420460,6 @@ xref: ICDO:9834/3 {source="NCIT:C4752"}
 xref: MedDRA:10042985 {source="Orphanet:86871", source="Orphanet:86871/e"}
 xref: MESH:D015461 {source="Orphanet:86871", source="MONDO:equivalentTo", source="Orphanet:86871/e"}
 xref: NCIT:C4752 {source="EFO:1000560", source="MONDO:equivalentTo"}
-xref: NCIT:C70649 {source="MONDO:equivalentObsolete"}
```

Comment:
`NCIT:C70649` is not obsolete.

Reply:
I am assuming that we will get `NCIT:C70649` back into Mondo during the alignment process.